production/historical_scrape/main_historical.R

```diff
@@ -0,0 +1,33 @@
+rm(list = ls())
+library(tidyverse)
+
+# read in configuration file to see what historical scrapers to run
+config_df <- read_csv(
+    "./production/historical_scrape/config.csv", col_types = cols())
```

Comment:
I think you changed the filename to `"./production/historical_scrape/historical_config.csv"` here.